tests/shopify_python/test_git_utils.py

```diff
@@ -248,3 +248,67 @@ def test_autopep_files(tmpdir):
     for fixed_file in files_to_autopep:
         with open(fixed_file) as file_to_check:
             assert file_to_check.readlines() == file_lines
+
+
+def test_linter(tmpdir):
+    # type: ('py.path.LocalPath') -> None
+    open(str(tmpdir.join('__init__.py')), 'w')
+
+    file_lines = [
+        "def foo():",
+        "  return 1\n"
+    ]
+    tmpdir.join('file.py').write('\n'.join(file_lines))
```

Comment:
I need to be explicit/exact with spacing here. The only way to use `"""` syntax would be to left-align all of the text flush with the left margin of the page. Otherwise `  return 1` doesn't start with 2 spaces, it starts with 2 + 2 indents * 4 spaces/indent = 10 spaces. That's not what I'm testing. Left-aligning to the edge of the page looks really ugly, so i don't want to do that either.

I'd like each line to be its own single-quote-delimited string, to make spacing explicit and clear. I can define it using single string concatenation, like this:
```
file_text = (
    "def foo():\n"
    "  return 1\n"
)
```
Or I can do:
```
file_lines = [
    "def foo():"
    "  return 1"
]
```
and use some variant of `writelines`, `'\n'.join(file_lines)`, or something similar.